.ci/jsoref-spellchecker/whitelist.words

```diff
@@ -23,6 +23,7 @@ AFDF
 AFilter
 aggregators
 Agoogle
+akmo
```

Comment:
Why we need this ?

Reply:
This was included by some rebasing error. Removed.